test/diagnostics/package-name-diagnostics.swift

```diff
@@ -3,11 +3,11 @@
 // Package name should not be empty
 // RUN: not %target-swift-frontend -typecheck %s -package-name "" 2>&1 | %FileCheck %s -check-prefix CHECK-EMPTY
 // CHECK-EMPTY: error: package-name is empty
-// CHECK-EMPTY: error: 'log' has a package access level but no -package-name was specified: {{.*}}.swift
+// CHECK-EMPTY: error: the package access level requires a package name; set it with the compiler flag -package-name
```

Comment:
Can we still pass the symbol name here? I usually find it helpful esp when there are multiple errors with the same diags.

Reply:
The tests don't show this but the diagnostics points to the attribute. Since the solution is to change a project setting I thought it would be enough. I can special case the check for a value decl if you prefer for us to keep the decl name.

Reply:
It's a nice to have, but not a strict requirement. I'm ok with adding it in the future if necessary.

Reply:
I added it back for ValueDecls.